__tests__/news.test.js

```diff
@@ -106,7 +102,77 @@ describe('GET /api/articles/:article_id', () => {
             expect(responseArray).toEqual("Not Found");
           })
     });
+    test('should return appropriate error if invalid input data type', () => {
+        return request(app)
+        .get(`/api/articles/TEXT`)
+        .expect(400)
+        .then((response) => {
+            const responseArray = response.res.statusMessage
+            expect(responseArray).toEqual("Bad Request");
+          })
+    });
+});
+
+describe('GET /api/articles', () => {
+    test('should return correct properties', () => {
+        return request(app)
+        .get(`/api/articles`)
+        .expect(200)
+        .then((response) => {
+            const responseArray = response.body.article
+            responseArray.forEach((article) => {
+                expect(Object.keys(article)).toHaveLength(8)
+                expect(article).toHaveProperty('author')
+                expect(article).toHaveProperty('title')
+                expect(article).toHaveProperty('article_id')
+                expect(article).toHaveProperty('topic')
+                expect(article).toHaveProperty('created_at')
+                expect(article).toHaveProperty('votes')
+                expect(article).toHaveProperty('article_img_url')
+                expect(article).toHaveProperty('comment_count')
+        })
```

Comment:
Nice. You may also want to check the types of the values. I.e. is topic a string, is votes a number etc.